internal/bridge/crunchybridgecluster/crunchybridgecluster_controller.go

```diff
@@ -415,6 +460,13 @@ func (r *CrunchyBridgeClusterReconciler) Reconcile(ctx context.Context, req ctrl
 	// Reconcile roles and their secrets
 	err = r.reconcilePostgresRoles(ctx, key, crunchybridgecluster)
 	if err != nil {
+		meta.SetStatusCondition(&crunchybridgecluster.Status.Conditions, metav1.Condition{
```

Comment:
I don't think we want this here... The reconciling of the roles could err out for reasons that have nothing to do with cluster status or our ability to communicate with the Bridge API.